flink-cdc-connect/flink-cdc-source-connectors/flink-connector-postgres-cdc/src/test/java/org/apache/flink/cdc/connectors/postgres/source/reader/PostgresSourceReaderTest.java

```diff
@@ -0,0 +1,75 @@
+package org.apache.flink.cdc.connectors.postgres.source.reader;
```

Comment:
Please add license and javadoc comment of class, nor compile will fails

Reply:
Yeap, added 👍